students/saleta_sebastian/lesson_01_basics/car_route.py

```diff
@@ -0,0 +1,11 @@
+def get_days(kilometers_per_day,route_length):
```

Comment:
Have a look at parameters names. Now you shadow 'kilometers_per_day' and 'route_length' from the outer scope. In rest of files, you have the same situation.